test/trento/application/integration/checks/checks_test.exs

```diff
@@ -80,7 +81,8 @@ defmodule Trento.Integration.ChecksTest do
           labels: "labels",
           name: "test 2",
           provider: :azure,
-          remediation: "remediation 2"
+          remediation: "remediation 2",
+          premium: nil
```

Comment:
Since we defined the property as `:boolean` I would use `false` here.
What is the reason we have in `nil`? :thinking:

Reply:
If we don't send the `premium` field in the incoming json, as it is not required, our code sets it as `nil`, as this is the default behaviour of our code (maybe inherited from ecto). 

I could try to make the false option the default one if the field is not coming though